egs/fisher_swbd/s5/local/chain/run_tdnn_7b.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=12
+train_stage=-10
+get_egs_stage=-10
+dir=exp/chain/tdnn_7b
+decode_iter=
+
+# TDNN options
+# this script uses the new tdnn config generator so it needs a final 0 to reflect that the final layer input has no splicing
+# smoothing options
+self_repair_scale=0.00001
+# training options
+num_epochs=4
+initial_effective_lrate=0.001
```

Comment:
A lot of these options can directly be passed to the train.py script. It does not have to be configurable. e.g. relu-dim etc. You could probably leave only num_epochs, remove_egs and common_egs_dir here.